CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.10.0-rc.1] - UNRELEASED
 
 ### Added
--
```

Comment:
replacing this line causes a conflict, it should just add a new one

Reply:
done